src/coreComponents/physicsSolvers/wavePropagation/pygeosx/main.py

```diff
@@ -0,0 +1,178 @@
+'''
+Created on 9/02/2021
+
+@author: macpro
+'''
+
+from functions import *
+import numpy as np
+from mesh import *
+from acquisition import *
+import pygeosx
+import sys
+from mpi4py import MPI
+from matplotlib import pyplot as plt
+from segy import *
+from print import *
+
+
+# Get the MPI rank
+comm = MPI.COMM_WORLD
+rank = comm.Get_rank()
+
+
+def main():
+    
+    #Initialize GEOSX
+    problem = pygeosx.initialize(rank, sys.argv)
+
+    pygeosx.apply_initial_conditions()
+    
+
+#========================================================
+#                 Initialization
+#========================================================
+
```

Comment:
This seems to share a lot in common with `functions.do_shots`.

Reply:
Yes the main.py was the previous version without the multiprocessing part. This file will no longer exist